### PR TITLE
simple message for empty objects tab in `Project-Overview`

### PR DESCRIPTION
**Pull request by laxels**

### CODE REVIEW

File: weave-js/src/components/Panel2/PanelRootBrowser/ProjectObjectsTable.tsx [hunk 1]
```diff
@@ -130,3 +131,22 @@ export const ProjectObjectsTable: React.FC<
...
+const EmptyState = memo(EmptyStateComp);
+
+const EmptyStateContainer = styled.div`
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const EmptyStateContent = styled.div``;
```
Comment (jamie-rasmussen) [inline weave-js/src/components/Panel2/PanelRootBrowser/ProjectObjectsTable.tsx:152]: optional: I feel like this isn't adding anything, maybe just use `<div>` above?

File: weave-js/src/components/Panel2/PanelRootBrowser/ProjectObjectsTable.tsx [hunk 2]
```diff
@@ -130,3 +131,22 @@ export const ProjectObjectsTable: React.FC<
     </PanelContextProvider>
   );
 };
+
+const EmptyStateComp: FC = () => {
```
Comment (jamie-rasmussen) [inline weave-js/src/components/Panel2/PanelRootBrowser/ProjectObjectsTable.tsx:135]: Any benefit to declaring this FC?

**Review by jamie-rasmussen — APPROVED** [review]

File: weave-js/src/components/Panel2/PanelRootBrowser/ProjectObjectsTable.tsx [hunk 3]
```diff
@@ -130,3 +131,22 @@ export const ProjectObjectsTable: React.FC<
...
+};
+
+const EmptyState = memo(EmptyStateComp);
+
+const EmptyStateContainer = styled.div`
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
```
[the remark this answers is not in the document]
Comment (laxels) [reply weave-js/src/components/Panel2/PanelRootBrowser/ProjectObjectsTable.tsx:140]: Good suggest. Added!

File: weave-js/src/components/Panel2/PanelRootBrowser/ProjectObjectsTable.tsx (hunk 1 above)
Comment (laxels) [reply weave-js/src/components/Panel2/PanelRootBrowser/ProjectObjectsTable.tsx]: Yeah, this was left over from when I thought we were gonna do something fancier than "No objects found`. Removed.

File: weave-js/src/components/Panel2/PanelRootBrowser/ProjectObjectsTable.tsx (hunk 2 above)
Comment (laxels) [reply weave-js/src/components/Panel2/PanelRootBrowser/ProjectObjectsTable.tsx]: Nope, same as above, I thought we were gonna do something more complicated. I've changed it to inlining `EmptyStateContainer` instead.

**Pull request merged**